src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -77,6 +77,18 @@ public class ConsoleOptions : CommandLineOptions
 
         #endregion
 
+        #region Overrides
+
+        protected override void CheckOptionCombinations()
+        {
+            base.CheckOptionCombinations();
+
+            if (RunAsX86 && ProcessModel == "InProcess")
+                ErrorMessages.Add("The --x86 and --inprocess options are incompatible.");
```

Comment:
That's a point since we can only run at 32-bit. In that case, there is no point in specifying it, however, so I would tend not to worry about it.